services/web/server/src/simcore_service_webserver/products/_db.py

```diff
@@ -42,6 +44,27 @@
 ]
 
 
+class PaymentFieldsTuple(NamedTuple):
+    enabled: bool
+    credits_per_usd: Decimal | None
+
+
+async def get_product_payment_fields(
+    conn: SAConnection, product_name: ProductName
+) -> PaymentFieldsTuple:
+    usd_per_credit = await get_product_latest_credit_price_or_none(
+        conn, product_name=product_name
+    )
+    if usd_per_credit is None or usd_per_credit == 0:
+        enabled = False
+        credits_per_usd = None
+    else:
+        enabled = True
+        credits_per_usd = Decimal(1) / usd_per_credit
```

Comment:
Ye, we need to double check how to setup/work properly with Decimal type when we are doing these kind of operations.

Reply:
so when we do the opposite operation, we do not end up with different value :)

Reply:
@matusdrobuliak66 I will delete the `products_price` database and add instead a column `credits_per_usd` in the `products` table.

Reply:
@pcrespov are you sure? I think products_price table has different purpose, for example to store historically prices. I like that we have that table. Also in hold additional information that otherwise you would need to create new columns in the products table. As mentioned already I think there is one to many relationship and the product_price has different purpose that product table.

Reply:
I think if you want to change the logic of division, just change it in the products_price table (drop column and create a new one)

Reply:
If you are referencing to our previous conversation, I still think we should introduce (in the future) the `billable` column in the `products` table. With that said, the `product_price` table has its purpose.